frontend/server/src/main/java/com/amazonaws/ml/mms/servingsdk_impl/ModelWorker.java

```diff
@@ -0,0 +1,25 @@
+package com.amazonaws.ml.mms.servingsdk_impl;
+
+import com.amazonaws.ml.mms.wlm.WorkerState;
+import com.amazonaws.ml.mms.wlm.WorkerThread;
+import software.amazon.ai.mms.servingsdk.Worker;
+
+public class ModelWorker implements Worker {
+    boolean isRunning;
+    long memory;
+
+    public ModelWorker(WorkerThread t) {
+        isRunning = t.getState() == WorkerState.WORKER_MODEL_LOADED;
+        memory = t.getMemory();
+    }
+
+    @Override
+    public Boolean getIsRunning() {
```

Comment:
boolean

Reply:
Done. I was handling null before. I modified hte code and forgot to change this to boolean.